packages/view/src/Slot.php

```diff
@@ -22,7 +22,7 @@ public function __construct(
     }
 
     public string $content {
-        get => base64_decode($this->content, true);
+        get => base64_decode($this->content, strict: true) ?: '';
```

Comment:
'0' -> encode -> decode -> ''

Reply:
```suggestion
        get => ($d = base64_decode($this->content, strict: true)) === false ? '' : $d;
```

Whoopsie. Would this be fine?

Reply:
> base64_decode returns either false or string, false casts to ''

We are using `declare(strict_types=1);` everywhere as a standard, so `false` will not cast to `''` in this case, but throw a `TypeError`, so we need to check the return type explicitly.

The `strict` param is still necessary, because the input can contain characters outside the base64 alphabet, we don't just want to discard those.

Reply:
strict_types are only related to implicit coercing of function input/output

But I took a look at a class as a whole, it doesn't have total control over $content, since it might be set by ::restore(). So everything is okay